Testscripts/Windows/PERF-XDP-TX-FORWARD.ps1

```diff
@@ -43,18 +43,18 @@ function Main {
         # Add modes according to testcase
         if ( $CurrentTestData.testName -match "TX" ){
             Add-Content -Value "modes=(xdp)" -Path $constantsFile
-            $packetCount=10000000
+            $testDuration=15

```

Comment:
This change is from measuring packet counts to testing time. Do you buy this change with DPDK folks?

Reply:
I kept time of 15 seconds coz at the rate of min 10MPPS we will get more than 150 Million packets sent. That's enough for testing packet forwarding.